### DISCUSSION TITLE
Windows afraid qutebrowser

### DISCUSSION THREAD

**Discussion by konstcode** [qutebrowser/qutebrowser#6171] [Ideas] [answered]
Hi guys, I loved your browser a lot. We need it a lot! :)
Windows precompiled version of qutebrowser starting from 2.0 have some problem with defender antivirus. He thinks that this some trojan(I hope this not :P). I don't believe windows defender, he always thinking bad about some opensource software, and I ignore his creams and restored exe from carantine.
This can works on my home pc but not on company laptop. With all those policies, I am not risking to do that :) And I wondering that it good to make it more pretty for windows. Also it will give to qutebrowser more popularity.

Thanks for doing your job. 👍

**Comment by The-Compiler** [comment, marked as answer]
See #6081 - if you can [report this to Microsoft](https://www.microsoft.com/en-us/wdsi/filesubmission) that'd help a lot, I'm a bit tired of doing that for every bigger release...
